Assets/MixedRealityToolkit/Inspectors/Setup/BuildDeployWindow.cs

```diff
@@ -0,0 +1,1436 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using Microsoft.MixedReality.Toolkit.WindowsDevicePortal;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml;
+using UnityEditor;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+using FileInfo = System.IO.FileInfo;
+
+namespace Microsoft.MixedReality.Toolkit.Build.Editor
+{
+    /// <summary>
+    /// Build window - supports SLN creation, APPX from SLN, Deploy on device, and misc helper utilities associated with the build/deploy/test iteration loop
+    /// Requires the device to be set in developer mode and to have secure connections disabled (in the security tab in the device portal)
+    /// </summary>
+    public class BuildDeployWindow : EditorWindow
+    {
+        #region Internal Types
+
+        private enum BuildDeployTab
+        {
+            UnityBuildOptions,
+            AppxBuildOptions,
+            DeployOptions
+        }
+
+        #endregion Internal Types
+
+        #region Constants and Readonly Values
+
+        private const string LOCAL_MACHINE = "Local Machine";
+
+        private const string HOLOLENS_USB = "HoloLens over USB";
+
+        private const string LOCAL_IP_ADDRESS = "127.0.0.1";
+
+        private const string EMPTY_IP_ADDRESS = "0.0.0.0";
+
+        private static readonly string[] TAB_NAMES = { "Unity Build Options", "Appx Build Options", "Deploy Options" };
+
+        private static readonly string[] SCRIPTING_BACKEND_NAMES = { "IL2CPP", ".NET" };
+
+        private static readonly int[] SCRIPTING_BACKEND_ENUMS = { (int)ScriptingImplementation.IL2CPP, (int)ScriptingImplementation.WinRTDotNET };
+
+        private static readonly string[] TARGET_DEVICE_OPTIONS = { "Any Device", "PC", "Mobile", "HoloLens" };
+
+        private static readonly string[] ARCHITECTURE_OPTIONS = { "x86", "x64", "arm" };
```

Comment:
This ends up looking a little odd: 
![image](https://user-images.githubusercontent.com/3580640/70934976-05529180-1ff4-11ea-81ac-e9a8ed5cb933.png)

since it's capitalized as `ARM` in other places, like Unity's build window:
![image](https://user-images.githubusercontent.com/3580640/70934999-14394400-1ff4-11ea-9af4-fb7fb0b1a889.png)

Reply:
Great find